int8 status
int16 tujuan_x
int16 tujuan_y